app/views/hat_requests/index.html.erb

```diff
@@ -38,14 +38,14 @@
         </div>
       <% end %>
 
-      <%= form_with model: hr, method: :delete do |f| %>
+      <%= form_with model: hr, method: :post do |f| %>
         <div class="boxline">
           <%= f.label :link, "Reason:", :class => "required" %>
           <%= f.text_area :reason, :rows => 4 %>
         </div>
         <p>
-            <%= f.submit "Approve Hat Request", formaction: update_hat_request_path(hr) %>
-            <%= f.submit "Reject Hat Request", class: "deletion", formation: destroy_hat_request_path(hr)  %>
+            <%= f.submit "Approve Hat Request", formaction: approve_hat_request_path(hr) %>
+            <%= f.submit "Reject Hat Request", class: "deletion", formaction: reject_hat_request_path(hr)  %>
```

Comment:
reverted "update" & "destroy" actions to "approve" & "reject" for clear intent. Also, there was a typo "formation" instead of "formaction". I'm unable to override delete and post request in those two forms.
Also, delete request doesn't make sense just in context of rejecting hat request as we're destroying hat request upon both approval & rejection.